src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -115,6 +117,7 @@ public HttpConnectionPool(HttpConnectionPoolManager poolManager, HttpConnectionK
             {
                 _originAuthority = new HttpAuthority(host, port);
 
+                // ToDo: what about this?
```

Comment:
This should be replaced with a check for version policy = exact and version = 3.0